source/shared/cpp/ObjectModel/ParseUtil.cpp

```diff
@@ -128,6 +128,33 @@ namespace AdaptiveSharedNamespace
         return propertyValue.asString();
     }
 
+    std::shared_ptr<BackgroundImage> ParseUtil::ExtractBackgroundImage(const Json::Value& json)
```

Comment:
>ExtractBackgroundImage [](start = 48, length = 22)

Maybe just GetBackgroundImage for consistency with the rest of the functions in this file